package.json

```diff
@@ -28,24 +28,20 @@
     "prettier:fix-all": "prettier --ignore-path .gitignore --write ."
   },
   "resolutions": {
-    "@material-ui/pickers@^3.2.10": "patch:@material-ui/pickers@npm%3A3.3.11#./.yarn/patches/@material-ui-pickers-npm-3.3.11-1c8f68ea20.patch",
-    "@material-ui/pickers@^3.3.10": "patch:@material-ui/pickers@npm%3A3.3.11#./.yarn/patches/@material-ui-pickers-npm-3.3.11-1c8f68ea20.patch",
     "@types/react": "^17",
-    "@types/react-dom": "^17",
-    "jest-haste-map@^29.7.0": "patch:jest-haste-map@npm%3A29.7.0#./.yarn/patches/jest-haste-map-npm-29.7.0-e3be419eff.patch"
+    "@types/react-dom": "^17"
   },
   "devDependencies": {
-    "@backstage/cli": "^0.26.10",
-    "@backstage/codemods": "^0.1.49",
-    "@backstage/create-app": "^0.5.16",
-    "@backstage/repo-tools": "^0.9.3",
+    "@backstage/cli": "^0.30.0",
+    "@backstage/codemods": "^0.1.52",
+    "@backstage/create-app": "^0.5.25",
+    "@backstage/repo-tools": "^0.13.0",
     "@changesets/cli": "^2.14.0",
     "@octokit/rest": "^19.0.3",
     "@playwright/test": "^1.32.3",
     "@spotify/eslint-config-typescript": "^15.0.0",
     "@spotify/eslint-plugin": "^14.1.3",
     "@spotify/prettier-config": "^14.0.0",
```

Comment:
Looks like this is no longer needed, replaced with @backstage/cli/config/prettier
https://backstage.github.io/upgrade-helper/?yarnPlugin=0&from=1.28.4&to=1.36.1